CONTRIBUTING.md

```diff
@@ -69,13 +69,19 @@ and pre-commit to run some validation checks:
 pipx install pre-commit  # or `pip install pre-commit` if you have a virtualenv
 ```
 
-You can optionally run the following command to run the validation checks as a git hook
+You can optionally install pre-commit hooks to automatically run the validation checks
 when making a commit:
 
 ```shell
 pre-commit install
 ```
 
+or you can run pre-commit manually as needed:
+
+```shell
+pre-commit run --all-files
```

Comment:
that's now duplicated with line 93|99

Reply:
Oh true. Do you think it's excessive?

Reply:
i'd remove either, i think it's important to keep the part that's required for a successful first PR as short as possible